### PR TITLE
Add a plugin to check platform name

### PR DESCRIPTION
**Pull request by pnuu**
This PR adds a plugin that aborts processing of the scene if the platrom name isn't in the list of processed platforms.

### CODE REVIEW

File: bin/satpy_launcher.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def process(msg, prod_list):
     job = message_to_job(msg, config)
     for wrk in config['workers']:
         cwrk = wrk.copy()
-        cwrk.pop('fun')(job, **cwrk)
+        try:
+            cwrk.pop('fun')(job, **cwrk)
+        except AbortProcessing:
+            LOG.info("Processing aborted for this scene")
```
Comment (mraspaud) [inline bin/satpy_launcher.py]: Can we add the error message to the info ?

Comment (pnuu) [reply bin/satpy_launcher.py]: In think the plugin should handle that, it has more info on what happened. Although, I'll add the name of the plugin to the message.

File: bin/satpy_launcher.py [hunk 2]
```diff
@@ -79,9 +79,10 @@ def process(msg, prod_list):
     for wrk in config['workers']:
         cwrk = wrk.copy()
         try:
-            cwrk.pop('fun')(job, **cwrk)
+            func = wrk('fun')
+            func(job, **cwrk)
```
Comment (mraspaud) [inline bin/satpy_launcher.py]: how does this work if `fun` isn't removed from `cwrk` ?

File: bin/satpy_launcher.py [hunk 3]
```diff
@@ -78,7 +78,11 @@ def process(msg, prod_list):
     job = message_to_job(msg, config)
     for wrk in config['workers']:
         cwrk = wrk.copy()
-        cwrk.pop('fun')(job, **cwrk)
+        try:
+            func = wrk('fun')
+            func(job, **cwrk)
+        except AbortProcessing:
+            LOG.info("Processing aborted by '%s'", func.__name__)
```
Comment (mraspaud) [inline bin/satpy_launcher.py]: don't we want to print the message too, ? like
```suggestion
            LOG.info("Processing aborted by '%s': %s", func.__name__, str(err))
```

File: bin/satpy_launcher.py (hunk 2 above)
Comment (pnuu) [reply bin/satpy_launcher.py]: I hoped it would? I'll add the `pop` in there.

File: bin/satpy_launcher.py (hunk 3 above)
Comment (pnuu) [reply bin/satpy_launcher.py]: The plugin should already have logged the reason, so don't think duplicate message is needed.

File: bin/satpy_launcher.py (hunk 2 above)
Comment (pnuu) [reply bin/satpy_launcher.py]: `pop` added

**Pull request merged**